File: app/src/main/java/com/blues/module_game/QuestionEntity.java
package com.blues.module_game;

import com.alibaba.fastjson.annotation.JSONField;

import java.io.Serializable;
import java.util.List;

public class QuestionEntity implements Serializable {


    @JSONField(name = "questions")
    public List<QuestionsBean> questions;

    public static class QuestionsBean implements Serializable {
        /**
         * id : 1
         * title : Question1
         * answer : A
         * selections : ["A","B","C","D"]
         */

        @JSONField(name = "id")
        public int id;
        @JSONField(name = "title")
        public String title;
        @JSONField(name = "answer")
        public String answer;
        @JSONField(name = "selections")
        public List<String> selections;
    }
}

File: app/src/main/java/com/blues/game/QuestionEntity.kt
package com.blues.game

import com.alibaba.fastjson.annotation.JSONField
import java.io.Serializable

class QuestionEntity : Serializable {

    @JSONField(name = "questions")
    var questions: List<QuestionsBean> = mutableListOf()

    class QuestionsBean : Serializable {

        /**
         * id : 1
         * title : Question1
         * answer : A
         * selections : ["A","B","C","D"]
         */
        @JSONField(name = "id")
        var id = 0

        @JSONField(name = "title")
        var title: String = ""

        @JSONField(name = "answer")
        var answer: String = ""

        @JSONField(name = "selections")
        var selections: List<String> = mutableListOf()
    }
}